README.md

```diff
@@ -1 +1,98 @@
-# angular-what-is-the-event-system-readme
\ No newline at end of file
+# What is the event system?
+
+## Overview
+
+Another powerful offering from Angular is the integrated event system. This allows us to publish and subcribe to events. We've actually used this already - remember using `$scope.$on` to listen to the `$destroy` event?
+
+## Objectives
+
+- Describe the publish/subscribe event pattern
+- Describe the event system
+
+## Publishing events
+
+All events can be published on our `$scope` or `$rootScope` objects.
+
+Angular offers us two ways of publishing events - either up or down. Up will go all the way from the current scope to our root scope, and down will go down from our current scope into it's children scopes, and all it's children's scopes, and so on and so forth.
```

Comment:
At this point in the README, i'm still a bit confused about child scopes. Add an intro section reminding the reader what you mean by child scopes.